packages/lib/getBrandColours.tsx

```diff
@@ -0,0 +1,193 @@
+  return `#${toHex(r)}${toHex(g)}${toHex(b)}`;
+}
+
+function lighten(hex: string, intensity: number): string {
+  const color = hexToRgb(`#${hex}`);
+
+  if (!color) {
+    return "";
+  }
+
+  const r = Math.round(color.r + (255 - color.r) * intensity);
+  const g = Math.round(color.g + (255 - color.g) * intensity);
+  const b = Math.round(color.b + (255 - color.b) * intensity);
+
+  return rgbToHex(r, g, b);
+}
+
+function darken(hex: string, intensity: number): string {
+  const color = hexToRgb(hex);
+
+  if (!color) {
+    return "";
+  }
+
+  const r = Math.round(color.r * intensity);
+  const g = Math.round(color.g * intensity);
+  const b = Math.round(color.b * intensity);
+
+  return rgbToHex(r, g, b);
+}
+
+function normalizeHexCode(hex: string | null, dark: boolean) {
+  if (!hex) {
+    return !dark ? BRAND_COLOR : DARK_BRAND_COLOR;
+  }
+
+  hex = hex.replace("#", "");
+
+  // If the length of the hex code is 3, double up each character
+  // e.g. fff => ffffff or a0e => aa00ee
+  if (hex.length === 3) {
+    hex = hex
+      .split("")
+      .map(function (hex) {
+        return hex + hex;
+      })
+      .join("");
+  }
+
+  return hex;
+}
+
+export const createColorMap = (brandColor: string) => {
+  const response: Record<string, string> = {
+    500: `#${brandColor}`.replace("##", "#"),
+  };
+  const intensityMap: {
+    [key: number]: number;
+  } = {
+    50: 0.95,
+    100: 0.9,
+    200: 0.75,
+    300: 0.6,
+    400: 0.3,
+    600: 0.9,
+    700: 0.75,
+    800: 0.6,
+    900: 0.49,
+  };
+
+  [50, 100, 200, 300, 400].forEach((level) => {
+    response[level] = lighten(brandColor, intensityMap[level]);
+  });
+
+  [600, 700, 800, 900].forEach((level) => {
+    response[level] = darken(brandColor, intensityMap[level]);
+  });
+  return response;
+};
+
+function getWCAGContrastColor(background: string): string {
+  // Convert the hex background color to RGB
+  const { r, g, b } = hexToRgb(background);
+  // Calculate the luminance of the background color
+  const luminance = (0.2126 * r + 0.7152 * g + 0.0722 * b) / 255;
+
+  // If the luminance is less than 0.5, use white as the text color, otherwise use black
+  return luminance < 0.5 ? "#FFFFFF" : "#000000";
+}
+
+/**
+ * Given a light and dark brand color value, update the css variables
+ * within the document to reflect the new brand colors.
+ */
+const useGetBrandingColours = ({
+  lightVal = BRAND_COLOR,
+  darkVal = DARK_BRAND_COLOR,
+}: {
+  lightVal: string | undefined | null;
+  darkVal: string | undefined | null;
+}) => {
+  const embedBrandingColors = useBrandColors();
+
+  lightVal = embedBrandingColors.brandColor || lightVal;
+
+  // convert to 6 digit equivalent if 3 digit code is entered
+  lightVal = normalizeHexCode(lightVal, false);
+
+  darkVal = normalizeHexCode(darkVal, true);
+
+  const lightColourMap = createColorMap(lightVal);
+  const darkColourMap = createColorMap(darkVal);
+
+  const theme = {
```

Comment:
Return a theme object that we use in 

`useCalcomTheme` to set these when brand colors change. 

This approach i believe was easier and nicer to use than the custom branding file. Had a discussion with @keithwillcode about it - happy to change back if anyone disagrees